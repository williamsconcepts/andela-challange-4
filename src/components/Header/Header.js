import React from 'react';
import classes from './Header.module.css';

const header = (props) => {
  const { heading } = props;
  return (
    <div className={`${classes.Header} tc`}>
      <h1 className={heading}>ESTIMATOR</h1>
    </div>
  );
};

export default header;
