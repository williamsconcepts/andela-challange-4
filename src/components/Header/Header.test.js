import React from 'react';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';
import Header from './Header';

expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));

describe('Test Header Component', () => {
  test('expect to render Header correctly', () => {
    expect.assertions(1);
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
