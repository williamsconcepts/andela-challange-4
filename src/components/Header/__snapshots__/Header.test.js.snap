// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header Component expect to render Header correctly 1`] = `
<div
  className="Header tc"
>
  <h1>
    ESTIMATOR
  </h1>
</div>
`;
